pilot/spec/tester_manager_spec.rb

```diff
@@ -113,110 +113,111 @@
-      it "creates a new tester and adds it to the default group" do
-        allow(current_user).to receive(:roles).and_return(["appmanager"])
-        allow(fake_client).to receive(:testers_by_app).and_return([])
-
-        expect(Spaceship::TestFlight::Tester).to receive(:create_app_level_tester).with(
-          app_id: 'com.whatever',
-          first_name: 'fake',
-          last_name: 'tester',
-          email: 'fabric-devtools@gmail.com+fake@gmail.com'
-        )
-        expect(Spaceship::Tunes::Tester::External).to receive(:find_by_app).and_return(nil) # before creating, no testers
-        expect(Spaceship::TestFlight::Group).to receive(:add_tester_to_groups!)
-        expect(Spaceship::Tunes::Tester::External).to receive(:find_by_app).and_return(fake_tester) # after creating a tester, we return one
-
-        expect(FastlaneCore::UI).to receive(:success).with('Successfully added tester: fabric-devtools@gmail.com+fake@gmail.com to app: My Fake App')
-        expect(FastlaneCore::UI).to receive(:success).with('Successfully added tester to the default tester group in app: My Fake App')
-
-        tester_manager.add_tester(default_add_tester_options)
-      end
-    end
-
-    describe "when admin asks to create new tester in the default group" do
-      it "creates a new tester and adds it to the default group" do
-        allow(current_user).to receive(:roles).and_return(["admin"])
-        allow(tester_manager).to receive(:find_app_tester).and_return(nil)
-
-        expect(Spaceship::Tunes::Tester::External).to receive(:create!).with(
-          email: 'fabric-devtools@gmail.com+fake@gmail.com',
-          first_name: 'fake',
-          last_name: 'tester'
-        )
-        expect(Spaceship::TestFlight::Group).to receive(:add_tester_to_groups!)
-        expect(FastlaneCore::UI).to receive(:success).with('Successfully added tester: fabric-devtools@gmail.com+fake@gmail.com to your account')
-        expect(FastlaneCore::UI).to receive(:success).with('Successfully added tester to the default tester group in app: My Fake App')
-
-        tester_manager.add_tester(default_add_tester_options)
-      end
-    end
-
-    describe "when admin asks to create new tester to a specific existing custom group" do
-      it "creates a new tester and adds it to the default group" do
-        allow(current_user).to receive(:roles).and_return(["admin"])
-        allow(tester_manager).to receive(:find_app_tester).and_return(fake_tester)
-
-        expect(Spaceship::Tunes::Tester::External).to_not receive(:create!)
-        expect(Spaceship::TestFlight::Group).to receive(:add_tester_to_groups!).and_return([custom_tester_group])
-        expect(FastlaneCore::UI).to receive(:success).with('Successfully added tester to group(s): Test Group in app: My Fake App')
-
-        tester_manager.add_tester(default_add_tester_options_with_group)
-      end
-    end
+    # describe "when invoked from a global context" do
```

Comment:
Why comment everything out?

Reply:
@milch I updated and added a couple back